### DISCUSSION TITLE
android app cannot render content。

### DISCUSSION THREAD

**Discussion by heartacker** [community/community#3972] [Mobile]
edge version

**Comment by elisealix22** [comment]
Hi @heartacker, I'm sorry you're running into this! Do you happen to have a screenshot of the issue that you could upload? Also, if you go to Settings and scroll to the bottom, can you confirm which app version you're using? That would really help us debug! Thank you so much.

**Reply by heartacker** [reply]
i can not read file using github app of android
 So that I can't upload an image.

**Reply by elisealix22** [reply]
Ah so this is happening when browsing repository files? Any steps you can list out to reproduce would be very helpful!

**Comment by heartacker** [comment]
1. Android app can't read file when i want to select a pic
![image](https://user-images.githubusercontent.com/1876302/118836821-cdd9ca00-b8f6-11eb-82d5-6cbd87bf5d5c.png)

2. this is the real content
![image](https://user-images.githubusercontent.com/1876302/118836918-e944d500-b8f6-11eb-9972-ed57dfd8735c.png)

3. and this is the android rendering, only show the frist line
![image](https://user-images.githubusercontent.com/1876302/118836716-b4d11900-b8f6-11eb-94d6-89b15170bc25.png)

**Reply by heartacker** [reply]
`v1.16.0-beta(10008)`

**Reply by heartacker** [reply]
githubAPP can't read pic when I select a picture

**Comment by Ahmed124323** [comment]
فيس بوك

**Comment by Dogscat3831** [comment]
Google chrome account

**Reply by Aguu171817k** [reply]
https://vm.tiktok.com/ZSRXmNwbW/

**Reply by Aguu171817k** [reply]
https://vm.tiktok.com/ZSRXm7ooE/

**Comment by ismailbahadir** [comment]
I'm sorry, I didn't know what kind of posts I was going to post from here, my aim was to try to unblock my account, I sent the message I wrote thinking you would send it to rapdy, I didn't understand what the posts were, I speak little English, I speak Turkish, sorry again

**Comment by heartacker** [comment]
it happen again
